test/integration/testdata/maptype/atomic-map.go

```diff
@@ -0,0 +1,7 @@
+package maptype
+
+// +k8s:openapi-gen=true
+type AtomicMap struct {
```

Comment:
Are these types supposed to generate something in some golden data? I don't see an update to some golden data

Reply:
Hm, good point. Not sure. The steps in the integration tests README don't change the golden data. The only thing that edits `golden.v2.json` is the `builder` script 🤔 I'll dig into it a bit as I don't know this codebase that well - let me know if it's something obvious I'm missing!

Reply:
I'm specifically concerned since the tests don't pass for me but they pass for the CI ...

Reply:
OK I can reproduce the failure on this pull-request while having master work.

Reply:
wait no it passes now ...

Reply:
Ahah it's absolutely terrible. You have to add your objects to:

https://github.com/kubernetes/kube-openapi/blob/master/test/integration/builder/main.go#L105-L117

We should have a way to fail if this is not done somehow ...

Reply:
Thanks! Added them with https://github.com/kubernetes/kube-openapi/pull/178/commits/a0384dd483d95391e59ea2171ff8c76ac4fb6c69 - though if I'm honest I only partially understand what they're testing 🤔

Reply:
The test generates the openapi corresponding to the types that you have. Now we can look at the `report.v2.json` and see that it generates what we expect from the tags.